Pipfile

```diff
@@ -95,6 +95,7 @@ utm = "*"
 websocket_client = "*"
 hatanaka = "==2.4"
 PySide2 = "*"
+networkx = "~=2.3"
```

Comment:
what's this for? is it also needed on agnos?

Reply:
for thneed, yeah can we get that in?